src/applications/vaos/utils/data.js

```diff
@@ -197,42 +197,14 @@ export function transformFormToAppointment(state) {
 
   return {
     clinic,
-    direct: {
```

Comment:
`direct` is taken care of by the api using the `desiredDate` and `dateTime` and `duration` passed in the root.